docs/source/plugins.rst

```diff
@@ -49,3 +49,26 @@ for more information on RabbitMQ and how to consume the messages.
        scheduler.add_plugin(plugin)
 
 Run with: ``dask-scheduler --preload <filename.py>``
+
+Accessing Full Task State
+-------------------------
+
+If you would like to access the full ``TaskState`` stored in the scheduler you
```

Comment:
For ease of linking

```suggestion
If you would like to access the full :ref:`distributed.scheduler.TaskTate` stored in the scheduler you
```

FYI @nicolls1, I was going to push this directly, but I don't want to mess up your master branch :) Typically you'd want to checkout a new branch from the latest master before starting on a PR.

Reply:
Good to know thanks! Sorry, was traveling and didn't have a chance to do it myself. Thanks for changing!